messaginginapp/src/sharedTest/java/io/customer/messaginginapp/ModuleMessagingInAppTest.kt

```diff
@@ -52,4 +53,28 @@ internal class ModuleMessagingInAppTest : BaseTest() {
         // verify given event listener gets registered
         verify(gistInAppMessagesProvider).setListener(eventListenerMock)
     }
+
+    @Test
+    fun initialize_givenComponentInitialize_expectGistToInitializeWithCorrectOrganizationId_expectGistToSetUserToken() {
```

Comment:
```suggestion
    fun initialize_givenProfilePreviouslyIdentified_expectGistToSetUserToken() {
```

I think a function name like this describes the test function better.